pkg/common/resource.go

```diff
@@ -159,8 +159,9 @@ func CreateUpdateRequestForNewNode(node Node) si.UpdateRequest {
 func CreateUpdateRequestForUpdatedNode(node Node) si.UpdateRequest {
 	// Currently only includes resource in the update request
 	nodeInfo := &si.UpdateNodeInfo{
-		NodeID:              node.name,
-		SchedulableResource: node.resource,
+		NodeID:               node.name,
+		Attributes:           make(map[string]string),
```

Comment:
This code has been the same for a long time and is only called within the mock scheduler (test only)
Currently without this change I do not see a nil pointer in the tests.

Please explain how this caused a nil pointer.